api/src/main/scala/com/gu/adapters/notifications/Notifications.scala

```diff
@@ -50,7 +57,7 @@ object Notifications {
     compactJson(renderJValue(Extraction.decompose(avatar)))
   }
 
-  def publishAvatar(publisher: Publisher, snsTopicArn: String, eventType: String, avatar: CreatedAvatar): Future[String] = {
+  def publishAvatar(publisher: Publisher, snsTopicArn: String, eventType: String, avatar: CreatedAvatar): Future[Unit] = {
```

Comment:
Driveby refactor, we're not using the result of this Future anywhere and returning a string doesn't make much sense.